packages/itwin/property-grid/src/components/FilteringPropertyGrid.tsx

```diff
@@ -46,6 +51,13 @@ export function FilteringPropertyGrid({ filterer, dataProvider, autoExpandChildC
     };
   }, [filterer, dataProvider, autoExpandChildCategories]);
 
+  const { value, inProgress } = useDebouncedAsyncValue(
```

Comment:
I think `value` should be renamed to `matchesCount`

Reply:
Renamed both `value` and `inProgress`.